Dockerfile-dev

```diff
@@ -0,0 +1,11 @@
+FROM node:11.13.0-alpine
+
+RUN mkdir -p /codebase
+WORKDIR /codebase
```

Comment:
Please use `src` dir instead of `codebase`

Reply:
ok